ui/package.json

```diff
@@ -20,13 +20,12 @@
     "precommit": "lint-staged"
   },
   "lint-staged": {
-    "gitDir": "../",
     "linters": {
-      "ui/{app,tests,config,lib,mirage}/**/*.js": [
-        "prettier-eslint --single-quote --trailing-comma es5 --print-width 110 --write",
+      "*.js": [
+        "prettier-eslint --single-quote --no-use-tabs --trailing-comma es5 --print-width 110 --write",
```

Comment:
❗️ Is there a story behind why `--no-use-tabs` was added? Could there be tabs in my files???